supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -39,6 +39,25 @@ Some tasks in this procedure require root privileges, which you can get temporar
 .Additional resources
 * link:https://www.redhat.com/sysadmin/difference-between-sudo-su[Exploring the differences between sudo and su commands in Linux]
 
+[[ellipses-in-yaml-code-blocks]]
+== Ellipses in YAML code blocks
+
+Use the pound sign (`#`) to comment out an ellipsis in YAML code blocks.
```

Comment:
```suggestion
Use the number sign (`#`) to comment out an ellipsis in YAML code blocks.
```

![Screenshot from 2024-10-07 09-24-31](https://github.com/user-attachments/assets/a934b85a-5a2e-495c-8184-5909d4d96782)

Reply:
I changed that and the commit did not appear. Will try again.